ocaml/xapi/certificates_sync.ml

```diff
@@ -0,0 +1,144 @@
+module D = Debug.Make (struct let name = "certificates_sync" end)
+
+open D
+module Unixext = Xapi_stdext_unix.Unixext
+module Date = Xapi_stdext_date.Date
+module G = Gencertlib
+module GP = Gencertlib.Pem
+module GV = Gencertlib.Validation
+open Api_errors
+open Rresult
+
+let ( let* ) = ( >>= )
+
+let is_tls_verification_enabled () =
+  Localdb.get_with_default Constants.tls_verification_enabled "false"
```

Comment:
Maybe it is worth checking here that the pool state (Db.Pool.tls_verification_enabled) matches the local state (what you are using here) - perhaps we should block installation in this case? Though this may be overzealous

Reply:
During boot we cant rely on the master connection - so I'd like to keep this local.